jobs/delorean/suites/integreatly/rc/branch.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- job:
+    name: delorean-suites/integreatly/rc/branch
+    display-name: 'integreatly-rc-branch'
+    project-type: pipeline
+    concurrent: false
+    parameters:
+      - string:
+          name: 'installationGitUrl'
+          default: 'git@github.com:integr8ly/installation.git'
+          description: '[REQUIRED] The installation repo'
+      - string:
+          name: 'suiteBranch'
+          default: 'integreatly-rc'
+          description: '[REQUIRED] The installation git branch to push new version changes'
+      - string:
+          name: 'productBranches'
+          default: '3scale-rc, amq-online-rc, codeready-rc, fuse-rc, fuse-online-rc, gitea-rc'
```

Comment:
I've removed our own products from this list as I don't believe we'll be having any RC for those. However, i'm not sure about codeready, fuse and gitea.

Reply:
@mikenairn @odra ^

Reply:
I'd probably just add them all tbh. It will do no harm as the job will just skip it if the branch doesn't exist.